python/ez_setup.py

```diff
@@ -1,284 +1,332 @@
-#!python
-
-# This file was obtained from:
-#   http://peak.telecommunity.com/dist/ez_setup.py
-# on 2011/1/21.
-
+#!/usr/bin/env python
```

Comment:
I'm sorry for what will be a somewhat strange request, but is there any way that this file can physically be located in a different directory? Specifically, can we put it in a "third_party" directory at the base of the protobuf repository?

The reason I hadn't updated this yet is because Google policy for open-source repos is that we aren't technically supposed to have this file here, since it's not Google-authored (or with a CLA).

Is there a way that's not too intrusive to do this here? Like just a little stub script that modifies sys.path, then imports the real ez_setup.py from ../third_party?